src/effects/native/echoeffect.cpp

```diff
@@ -26,26 +27,28 @@ EffectManifest EchoEffect::getManifest() {
 
     EffectManifestParameter* delay = manifest.addParameter();
     delay->setId("delay_time");
-    delay->setName(QObject::tr("Delay"));
```

Comment:
I think "Delay" is confusing because that often refers to a more complex effect:

> The term “echo” was used more often in the early days, and is sometimes used today to refer to the distinct and distant repeats of a signal, while “delay” refers to anything from the same, to the short repeats heard as reverb, to the complex, long, manipulated repeats of an intricate digital delay line.

http://www.gibson.com/News-Lifestyle/Features/en-us/effects-explained-echo-delay.aspx